### DISCUSSION TITLE
Благодарность и xvideos

### DISCUSSION THREAD

**Discussion by Konserva228** [dovecoteescapee/ByeDPIAndroid#137] [Q&A]
Установил плод ваших замечательных трудов и все заработало на мобильном Мегафоне. Правда пришлось поискать специфические настройки именно для Мегафона в теме по проблемам с Ютубом. Большое вам спасибо за ваши волшебные умственные способности, дорогие мои борцы за свободу интернета!

Но подскажите вот такой момент. Доступ появился буквально везде: и западные соц.сети, и торренты, и даже порнхаб. Но почему не открывается xvideos.com (не то чтобы я особый любитель ;) , но хочется до конца уже разобраться)? Он не добавлен в список для обхода как-то с вашей стороны (извините если этот вопрос тупой. Я ламер...)? И можно ли этот ресурс добавить как-нибудь для обхода?

**Comment by Vstalker72** [comment]
Он заблокирован по IPшнику

> Установил плод ваших замечательных трудов и все заработало на мобильном Мегафоне. Правда пришлось поискать специфические настройки именно для Мегафона в теме по проблемам с Ютубом. Большое вам спасибо за ваши волшебные умственные способности, дорогие мои борцы за свободу интернета!
> 
> Но подскажите вот такой момент. Доступ появился буквально везде: и западные соц.сети, и торренты, и даже порнхаб. Но почему не открывается xvideos.com (не то чтобы я особый любитель ;) , но хочется до конца уже разобраться)? Он не добавлен в список для обхода как-то с вашей стороны (извините если этот вопрос тупой. Я ламер...)? И можно ли этот ресурс добавить как-нибудь для обхода?
